java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -630,6 +634,34 @@ public CompletableFuture<String[]> lpopCount(@NonNull String key, long count) {
                 response -> castArray(handleArrayResponse(response), String.class));
     }
 
+    @Override
+    public CompletableFuture<Long> lpos(@NonNull String key, @NonNull String element) {
+        return commandManager.submitNewCommand(
+            LPos, new String[] {key, element}, this::handleLongResponse);
+    }
+
+    @Override
+    public CompletableFuture<Long> lpos(@NonNull String key, @NonNull String element, @NonNull LPosOptions options) {
+        return commandManager.submitNewCommand(
+            LPos, new String[] {key, element, Arrays.toString(options.toArgs())}, this::handleLongResponse);
+    }
+
+    @Override
+    public CompletableFuture<Long[]> lposCount(@NonNull String key, @NonNull String element, long count) {
+        return commandManager.submitNewCommand(
+            LPos,
+            new String[] {key, element, Long.toString(count)},
+            response -> castArray(handleArrayResponse(response), Long.class));
+    }
+
+    @Override
+    public CompletableFuture<Long[]> lposCount(@NonNull String key, @NonNull String element, long count, @NonNull LPosOptions options) {
+        return commandManager.submitNewCommand(
+            LPos,
+            new String[] {key, element, Long.toString(count), Arrays.toString((options.toArgs()))},
+            response -> castArray(handleArrayResponse(response), Long.class));
+    }
```

Comment:
You wouldn't need these two functions if you add `count` as another `Long` class variable in `LPosOptions` like how you did for `rank` and `maxLength`

Reply:
Nvm, I missed that it has a different return type.